pkg/xen-tools/Dockerfile

```diff
@@ -60,9 +59,8 @@ WORKDIR /
 
 # Download and verify xen
 #TODO: verify Xen
-RUN \
-    [ -f "$(basename ${XEN_SOURCE})" ] || curl -fsSLO "${XEN_SOURCE}" && \
-    tar --absolute-names -xz < "$(basename ${XEN_SOURCE})" && mv "/xen-${XEN_VERSION}" /xen
```

Comment:
As I understand the logic here was about possibility to use the file located in the directory instead of downloading (to try own bundle). Do we want to keep this feature (we have more clear approach with temp file in pkg/kernel)?

Reply:
Yeah, I realized that with grub after this one was done. We can put it back.